src/schemas/Helpdesk/gqlTypeDefs.ts

```diff
@@ -0,0 +1,38 @@
+import { gql } from 'apollo-server-express';
+
+export default gql`
+  enum Category {
```

Comment:
I would name this someting like `JiraTicketCategory`, only because when this gets merged into the larger schema, `Ticket` is a little generic and creates room for namespace conflict. 
It's an unfortunate thing that we can't really do anonymous types or namespace them in GraphQl so we have this tricky split situation eh